single_tech_samples/streamanalytics/test/testConfig.json

```diff
@@ -34,8 +34,8 @@
       "ExpectedOutputs": [
         {
           "OutputAlias": "bloboutput",
-          "FilePath": "/dev/null",
-          "Required": true
+          "FilePath": "temperature_less_than_27_degrees.json",
+          "Required": false
```

Comment:
What was the motivation for this change? This looks like it will cause the test to pass when it should fail

Reply:
So the problem is that when this is true, there's errors that there's no "bloboutput". 
![image](https://user-images.githubusercontent.com/9697560/196852636-eacf195d-1e90-41f4-b0bb-d72e0fa72946.png)

Reply:
I think this may be a bug with the tool, so I've reverted my change